packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Map/__mocks__/loadGoogleMapsApi.js

```diff
@@ -0,0 +1,43 @@
+const mocks = {
+    googleMapsInstance: {
+        fitBounds: jest.fn(),
+        setCenter: jest.fn(),
+        setZoom: jest.fn()
+    },
+    googleMapsMarkerInstance: {
+        addListener: jest.fn()
+    },
+    googleMapsLatLngBoundsInstanceExtend: jest.fn(),
+    googleMapsInfoWindowInstanceOpen: jest.fn(),
+    googleMapsInfoWindowInstanceClose: jest.fn(),
+    GoogleMapsLatLngBoundsConstructor: jest.fn(() => ({
+        extend: mocks.googleMapsLatLngBoundsInstanceExtend
+    })),
+    GoogleMapsInfoWindowConstructor: jest.fn(() => ({
+        open: mocks.googleMapsInfoWindowInstanceOpen,
+        close: mocks.googleMapsInfoWindowInstanceClose
+    }))
+};
+
+mocks.googleMapsInstance = {
```

Comment:
Isn't this already part of the mock from the const above?

Reply:
haha 💯